test/test_validate_explanations.py

```diff
@@ -38,7 +38,7 @@ def test_verify_pipeline_model_coefficient_explanation(self):
         # Note: in pipeline case, we use KernelExplainer;
         # in linear case we use LinearExplainer which is much faster
         pipeline = [True, False]
-        threshold = [0.85, 0.76]
+        threshold = [0.62, 0.60]
```

Comment:
Yikes, that's a big hit

Reply:
recommend pinning the default to where it was before instead of the accuracy change

Reply:
done!